File: przykladowe.py
# Print basic info about clients into stderr
import sys

def emp(entity):
	if entity.employed:
		return "employed"
	return "unemployed"

def mar(entity):
	if entity.married:
		return "married"
	return "single"

for e in entities:
    sys.stderr.write(e.name + " " + e.surname + " - age: "+str(e.age)+", "+emp(e)+" "+mar(e)+"\n")

# Swap names and surnames of clients
for e in entities:
    e.name, e.surname = e.surname, e.name

File: przykladowe.js
// Podaj punkty zdolności kredytowej dla klientów

for each(var e in entities) {
	var score = 0;

	if(e.employed) {
		score += 100 - e.age;
		if(e.married) {
			score += 100;
		}
		score += e.monthSalary / 10 - 100;
		score -= e.numOfChildren * 10;
	}

	print(e.name + " " + e.surname + ": " + score);
}

// Uaktualnij wiek każdego z klientów

for each(var e in entities) {
    e.age++
}

